cfxdb/__init__.py

```diff
@@ -5,6 +5,9 @@
 #
 ##############################################################################
 
+import txaio
+txaio.use_twisted()  # noqa
```

Comment:
@oberstet this change on the tin looks problematic because it "limits" the library to have a hard dependency on Twisted. Its a bit odd as to why pytest imports the main module first instead of the test module itself.

Reply:
Without that change, all tests were failing https://travis-ci.com/github/crossbario/cfxdb/jobs/358370868

Reply:
> this change on the tin looks problematic because it "limits" the library to have a hard dependency on Twisted. 

oh, you're right, agreed. we should revisit and fix that at some point